src/openrct2-ui/windows/TitleMenu.cpp

```diff
@@ -31,10 +31,10 @@ enum {
 };
 
 static rct_widget window_title_menu_widgets[] = {
-    { WWT_IMGBTN, 2, 0, 0, 0, 81, SPR_MENU_NEW_GAME,        STR_START_NEW_GAME_TIP          },
-    { WWT_IMGBTN, 2, 0, 0, 0, 81, SPR_MENU_LOAD_GAME,       STR_CONTINUE_SAVED_GAME_TIP     },
-    { WWT_IMGBTN, 2, 0, 0, 0, 81, SPR_G2_MENU_MULTIPLAYER,  STR_SHOW_MULTIPLAYER_TIP        },
-    { WWT_IMGBTN, 2, 0, 0, 0, 81, SPR_MENU_TOOLBOX,         STR_GAME_TOOLS_TIP              },
+    MakeWidget({0, 0}, {1, 82}, WWT_IMGBTN, 2, SPR_MENU_NEW_GAME,       STR_START_NEW_GAME_TIP),
+    MakeWidget({0, 0}, {1, 82}, WWT_IMGBTN, 2, SPR_MENU_LOAD_GAME,      STR_CONTINUE_SAVED_GAME_TIP),
+    MakeWidget({0, 0}, {1, 82}, WWT_IMGBTN, 2, SPR_G2_MENU_MULTIPLAYER, STR_SHOW_MULTIPLAYER_TIP),
+    MakeWidget({0, 0}, {1, 82}, WWT_IMGBTN, 2, SPR_MENU_TOOLBOX,        STR_GAME_TOOLS_TIP),
```

Comment:
Hmm, these looks odd to me with a width of 1. They're probably set in an invalidate event, but I think we should adjust the definitions while we're at it.

Reply:
This is so because the third widget, for the multiplayer, can sometimes be disabled:
https://github.com/OpenRCT2/OpenRCT2/blob/01d0287ee2a135e8f337c75df4834f92da23093c/src/openrct2-ui/windows/TitleMenu.cpp#L91-L96
Thus, it does this logic to calculate the coordinates:
https://github.com/OpenRCT2/OpenRCT2/blob/01d0287ee2a135e8f337c75df4834f92da23093c/src/openrct2-ui/windows/TitleMenu.cpp#L99-L114

I'm a bit confuse as to whether we should fix it above. It will look mostly alright, but we will still have to keep the function above to readjust the positions in case the multiplayer is disabled.

Reply:
Thanks — yes, that explains it. I propose we change it to the following:
```suggestion
    MakeWidget({0, 0}, {82, 82}, WWT_IMGBTN, 2, SPR_MENU_NEW_GAME,       STR_START_NEW_GAME_TIP),
    MakeWidget({0, 0}, {82, 82}, WWT_IMGBTN, 2, SPR_MENU_LOAD_GAME,      STR_CONTINUE_SAVED_GAME_TIP),
    MakeWidget({0, 0}, {82, 82}, WWT_IMGBTN, 2, SPR_G2_MENU_MULTIPLAYER, STR_SHOW_MULTIPLAYER_TIP),
    MakeWidget({0, 0}, {82, 82}, WWT_IMGBTN, 2, SPR_MENU_TOOLBOX,        STR_GAME_TOOLS_TIP),
```

This way, the dimensions of the buttons are clear, even if their coordinates will be changed. Once the underlying widget structure has been changed, we can simplify the code that repositions them, too.

Reply:
Done.